.gitlab-ci.yml

```diff
@@ -96,7 +96,7 @@ check:test-generate:
     - if: $CI_COMMIT_BRANCH && $CI_COMMIT_BRANCH !~ /^(?:master|staging)$/ && $CI_COMMIT_TITLE !~ /^[0-9]+\.[0-9]+\.[0-9]+(?:-.*[0-9]+)?$/
       when: manual
 
-check:test:
+.check:test:
```

Comment:
Can you re-enable check tests for this PR. I want to see the CI/CD run tests over this.

Reply:
Re-enabled